test/test_consumer_integration.py

```diff
@@ -88,6 +94,12 @@ def kafka_consumer(self, **configs):
                                  **configs)
         return consumer
 
+    def kafka_producer(self, **configs):
+        brokers = '%s:%d' % (self.server.host, self.server.port)
+        producer = KafkaProducer(
+            bootstrap_servers=brokers, **configs)
+        return producer
```

Comment:
Understood. You can apply pytest fixtures at the class level in pytest, but only if it's a generic class, aka designed for pytest and not unittest.